spec/02-integration/04-admin_api/02-apis_routes_spec.lua

```diff
@@ -259,6 +315,55 @@ describe("Admin API #" .. kong_config.database, function()
         end
       end)
       describe("errors", function()
+        it("returns 415 on invalid content-type", function()
+          local res = assert(client:request {
+            method = "POST",
```

Comment:
Are those tests duplicates of the above ones? Maybe the intent was to re-run them with `PUT`?

Reply:
Yes, a copy-paste-bug. Pushed a new version.